app/serializers/user_card_serializer.rb

```diff
@@ -222,6 +223,10 @@ def card_background_upload_url
     object.card_background_upload&.url
   end
 
+  def status
```

Comment:
Anything we need to consider here https://github.com/discourse/discourse/blob/main/app/controllers/users_controller.rb#L108 ?

Reply:
This method returns status without additional changes (I've just tested it). I wonder if you know where we use this method? Quick search on the client-side do not return any usage.

Reply:
I was more concerned about maybe having to preload something to query in the serializer

Reply:
Also you are right I don't know where user-cards or cards is used 🤔  I can see @tgxworld made specific changes and tests in discourse-follow to not break this route, maybe he knows?

Reply:
Anyway, preloading won't hurt here, so I've added it (still curious where we use it, hope tgxworld know)

Reply:
It was introduced in https://github.com/discourse/discourse/pull/9078 so maybe @davidtaylorhq remember something.

Reply:
The multi-user-card route was added for https://meta.discourse.org/t/user-card-directory/144479?u=david (Internal ref https://dev.discourse.org/t/24089)

Reply:
So a core route only used in one theme, well 🤷‍♂️  At least we have the answer :p thanks